addon/v1alpha1/types_clustermanagementaddon.go

```diff
@@ -74,14 +74,18 @@ type ConfigCoordinates struct {
 	// +optional
 	LastObservedGeneration int64 `json:"lastObservedGeneration,omitempty"`
 
-	// configGroupResource represents the GroupResource of the add-on configuration.
+	// configGroupResources is a list of configuration types supported by add-on.
+	// An empty list means the add-on does not require configurations.
+	// The default is an empty list
 	// +optional
-	ConfigGroupResource *ConfigGroupResource `json:"configGroupResource,omitempty"`
+	ConfigGroupResources []ConfigGroupResource `json:"configGroupResources,omitempty"`
```

Comment:
seems this is duplicated with DefaultConfigs

Reply:
yes. it is, I will try to refactor this part